bentoml/marshal/marshal.py

```diff
@@ -141,6 +140,13 @@ def __init__(
         outbound_workers=1,
         mb_max_batch_size: int = None,
         mb_max_latency: int = None,
+        request_header_flag: str = Provide[
+            BentoMLContainer.config.marshal_server.request_header_flag
```

Comment:
use `from bentoml.configuration.containers import BentoMLContainer as BTC` to make it shorter? 🤔🤔🤔

Reply:
Elon Must would agree. :)